src/thunderbird_accounts/subscription/tasks.py

```diff
@@ -199,9 +201,56 @@ def paddle_subscription_event(self, event_data: dict, occurred_at: datetime.date
             'current_billing_period_starts_at': current_billing_period_starts_at,
             'current_billing_period_ends_at': current_billing_period_ends_at,
             'user_id': user_uuid,
+            'webhook_updated_at': occurred_at,
         },
     )
 
+    # If a transaction_id is included, then update/create the association to this subscription
+    if transaction_id:
+        Transaction.objects.update_or_create(
+            paddle_id=transaction_id,
+            defaults={
+                'paddle_subscription_id': paddle_id,
+                'subscription_id': subscription.uuid,
+            },
+        )
+
+    for item in subscription_items:
+        # Note: These are reoccurring items only
+        # Slurp up the subscription items, these are only created if all three items are new
+
+        quantity = item.get('quantity')
+        price = item.get('price')
+        product = item.get('product')
+        price_obj = None
+
+        # If we have the data we can create a price object now and reference in the subscription item
+        if price:
+            unit_price = price.get('unit_price', {})
+            billing_cycle = price.get('billing_cycle', {})
+            price_obj, price_obj_created = Price.objects.update_or_create(
+                paddle_id=price.get('id'),
+                defaults={
+                    'paddle_product_id': product.get('id'),
+                    'name': price.get('name'),
+                    'amount': unit_price.get('amount'),
+                    'currency': unit_price.get('currency_code'),
+                    'price_type': price.get('type'),
+                    'billing_cycle_frequency': billing_cycle.get('frequency'),
+                    'billing_cycle_interval': billing_cycle.get('interval'),
+                    'webhook_updated_at': occurred_at,
+                },
+            )
```

Comment:
Instead of handling price webhooks (which we may still do..) we get the full info within the subscription items, so might as well store them.